src/mavlink/mavlink_camera.rs

```diff
@@ -320,30 +319,11 @@ fn receive_message_loop(
                                 debug!("Sending video_stream_information..");
                                 let information =
                                     mavlink_camera_information.as_ref().lock().unwrap();
-                                let source_string =
-                                    information.video_source_type.inner().source_string();
-
-                                // Remove localhost address with public ip
-                                let mut video_url = information.video_stream_uri.clone();
-                                if let Ok(address) = std::net::Ipv4Addr::from_str(
-                                    video_url.host_str().unwrap_or_default(),
-                                ) {
-                                    if address == std::net::Ipv4Addr::UNSPECIFIED {
```

Comment:
I'm not following the changes on this commit, where is the logic to update the host address accessible by QGC ?

Reply:
I've changed this a little now, it is a method [here](https://github.com/bluerobotics/mavlink-camera-manager/pull/107/files#diff-2312b8ed03f8924e5bb2b3c73c9d98aac348561e8e9d0ac77c5457a9d730b89eR210).